flaskbb/templates/forum/topic.html

```diff
@@ -138,6 +138,12 @@
                         </form>
                         {% endif %}
                     {% endif %}
+					{% if current_user|can_moderate(topic.forum) %}
+						<form class="inline-form" method="post" action="{{ url_for('management.ban_user', user_id = user.id) }}">
```

Comment:
This is an OCD thing but it looks like the beginning <form> tag doesn't match up with the ending one.